### ISSUE TITLE
Add a job button does not work in glpi 9.2.x

### ISSUE THREAD

**Issue by OfficeLink** [fusioninventory/fusioninventory-for-glpi#2837]
Hello, I'm trying to create a job for a task, but when I click the button nothing happens.

The same occurs in versions GLPI 9.4.2 and 9.4.3

Thank you.

![Not-Create-Job](https://user-images.githubusercontent.com/52462110/60526352-96dc7a00-9cc6-11e9-8c3e-bc7216d0a929.jpg)

**Comment by 77oleg** [comment]
Same problem here and can't still solve. There is no errors in logs or debugging console.

**Comment by 77oleg** [comment]
There is something wrong with these checks in config.php:
Commenting out lines below makes Add job button working.

//      if (script_endswith("task.form.php")
//         || script_endswith("taskjob.php")
//         || script_endswith("iprange.form.php")) {
         array_push(
            $PLUGIN_HOOKS['add_javascript']['fusioninventory'],
            "lib/lazy.js-0.5.1/lazy".($debug_mode?"":".min").".js",
            "lib/mustache.js-2.3.0/mustache".($debug_mode?"":".min").".js",
            "js/taskjobs".($debug_mode || !file_exists('js/taskjobs.min.js')?"":".min").".js"
         );
//      }

**Comment by 3WTECNOLOGIA** [comment]
Obrigado!

Não localizei o arquivo config.php.....

Pode me dizer qual o caminho?

**Comment by OfficeLink** [comment]
> There is something wrong with these checks in setup.php:
> Commenting out lines below makes Add job button working.
> 
> // if (script_endswith("task.form.php")
> // || script_endswith("taskjob.php")
> // || script_endswith("iprange.form.php")) {
> array_push(
> $PLUGIN_HOOKS['add_javascript']['fusioninventory'],
> "lib/lazy.js-0.5.1/lazy".($debug_mode?"":".min").".js",
> "lib/mustache.js-2.3.0/mustache".($debug_mode?"":".min").".js",
> "js/taskjobs".($debug_mode || !file_exists('js/taskjobs.min.js')?"":".min").".js"
> );
> // }


Yes, commenting on these lines and those in # 2831 solved the problem and now the button works normally.
Thank you!!

**Comment by OliQuentDigital** [comment]
Hello, 

Here i have this in my file, it seems that it's already commenting out but the Add Job Button is still not working
